### PR TITLE
Show which keyword argument is repeated on 3.9+

### PR DESCRIPTION
**Pull request by isidentical**

### CODE REVIEW

File: test/test_python_errors.py [hunk 1]
```diff
@@ -359,6 +359,10 @@ def test_continue_in_finally():
...
 )
 def test_forbidden_name(template, target):
-    assert _get_error_list(template.format(target=target), version="3")[0].message
+    assert _get_error_list(template.format(target=target), version="3")
 
 
+def test_repeated_kwarg():
+    # python 3.9+ shows which argument is repeated
+    assert "q" not in _get_error_list("f(q=1, q=2)", version="3.8")[0].message
+    assert "q" in _get_error_list("f(q=1, q=2)", version="3.9")[0].message
```
Comment (davidhalter) [inline test/test_python_errors.py:368]: Can you assert the full messages here? I think it's easier to read the test that way.

Thanks!

Comment (isidentical) [reply test/test_python_errors.py:368]: Sure!

**Comment by davidhalter** [comment]
Consider this merged when the CI passes. It's just that the CI doesn't start.

But I learned that I should wait with merging until the CI passes :smile:

**Comment by isidentical** [comment]
> But I learned that I should wait with merging until the CI passes smile

Hahaha, indeed

**Comment by isidentical** [comment]
By the way, I dont know if you still use the email in your github profile but I sent an email there. It is not that important, but it would be awesome for me  if you can check it out.

**Comment by davidhalter** [comment]
> By the way, I dont know if you still use the email in your github profile but I sent an email there. It is not that important, but it would be awesome for me if you can check it out.

Yeah, I started answering yesterday, but I'm still thinking about it. Will try to send it to you, today. :)

**Pull request merged**